### PR TITLE
chore: replace deprecated function

### PR DESCRIPTION
**Pull request by yyzxw**
replace instr.FromInt to instr.FormInt32, FromInt has been deprecated

### CODE REVIEW

File: internal/xds/v3/contour.go
Comment (davinci26) [inline internal/xds/v3/contour.go:85]: can we revert this change since this unrelated to the goal of this PR? I am worried that if there is something wrong here it would be hard to parse through the commit history

**Review by davinci26 — COMMENTED** [review]
Thanks for the cleanup! Appreciate it

I am ok with both the `docs` change and the `replacement` in a single PR but I would defer to the maintainers to make the final call. 

If we decide to keep take the PR as is tho we should update the title and PR description such that it is clear what this PR does.

File: internal/xds/v3/contour.go
Comment (yyzxw) [reply internal/xds/v3/contour.go]: done

**Review by sunjayBhatia — CHANGES_REQUESTED** [review]
I believe these comments from a previous iteration of this PR still apply to this one:
- https://github.com/projectcontour/contour/pull/5755#pullrequestreview-1631780001
- https://github.com/projectcontour/contour/pull/5755#issuecomment-1790986012

**Pull request closed without merging**
